timeless/message/property_resource.py

```diff
@@ -0,0 +1,19 @@
+""" Message resource for internationalization based on a property file """
+
+from timeless.message.message_resource import MessageResource
+
+
+class PropertyResource(MessageResource):
+    """ Resources read from a property file.
+        @todo #269:30min Implement FileResource. FileResource should read a
+         property file and return the value according to the keys. Locale
+         should be used to define from which locale the file belongs. The
+         locale is defined in the fine name, for example,
+         messages_en_US.properties. An exception must be raised if some
```

Comment:
@paulodamaso I wouldn't throw an exception if a translation is missing, much better to return a default string (for example "Translation not found: <key>"

Reply:
@emilianodellacasa I think that throwing an exception would adhere better to the "fail fast" philosophy: our program breaks if we do not have the translation, we know about the error, fix it and all becomes ok. If we avoid breaking the code here we could just mask the problem and we it can't be detected until too late. See if this https://www.yegor256.com/2015/08/25/fail-fast.html makes sense.